src/app/unlocked-stage/unlocked-stage.component.ts

```diff
@@ -41,11 +43,11 @@ export class UnlockedStageComponent implements OnInit {
   private mosquitoes = [];
   private maxMosquitoes = 10;
   private mosquitoesStrength = Array(this.maxMosquitoes).fill(100);
-  private mosCountInitial = 2;
-  private mosCountCurrent = this.mosCountInitial;
+  private mosCountCurrent: number;
   private keydown = false;
   private levelCounter = 1;
-  private maxLevel = 4;
+  private level: number;
+  public currStage = 10;
```

Comment:
constant here..

Reply:
currStage is a constant right ?  Please resolve issues in this PR first so it won't cause problems.. Its been open since ages.